_posts/2015-12-18-Rails-5-0-beta1.markdown

```diff
@@ -0,0 +1,86 @@
+
+You should really checkout the CHANGELOGs, though. There's just so much new and good stuff available in all the frameworks:
+
+* [Action Mailer CHANGELOG](https://github.com/rails/rails/blob/v5.0.0.beta1/actionmailer/CHANGELOG.md)
+* [Action Pack CHANGELOG](https://github.com/rails/rails/blob/v5.0.0.beta1/actionpack/CHANGELOG.md)
+* [Action View CHANGELOG](https://github.com/rails/rails/blob/v5.0.0.beta1/actionview/CHANGELOG.md)
+* [Active Model CHANGELOG](https://github.com/rails/rails/blob/v5.0.0.beta1/activemodel/CHANGELOG.md)
+* [Active Record CHANGELOG](https://github.com/rails/rails/blob/v5.0.0.beta1/activerecord/CHANGELOG.md)
+* [Active Support CHANGELOG](https://github.com/rails/rails/blob/v5.0.0.beta1/activesupport/CHANGELOG.md)
+* [Active Job CHANGELOG](https://github.com/rails/rails/blob/v5.0.0.beta1/activejob/CHANGELOG.md)
+* [Railties CHANGELOG](https://github.com/rails/rails/blob/v5.0.0.beta1/railties/CHANGELOG.md)
+
+Claudio did a [nice little slide deck][claudio-slides] walking through some of his favorite improvements.
+
+Note too that we're cooking Turbolinks 5 – the one with native iOS and Android wrapper implementations! – for concurrent release with Rails 5. You can follow along on [basecamp/turbolinks/v5][turbolinks-v5].
+
+## Maintenance consequences
+
+As per our [maintenance policy][maintenance-policy], the release of Rails 5.0 will mean that bug fixes will only apply to 5.0.x, regular security issues to 5.0.x and 4.2.x, and severe security issues also to 5.0.x and 4.2.x (but when 5.1 drops, to 5.1.x, 5.0.x, and 4.2.x). This means 4.1.x and below will essentially be unsupported! Ruby 2.2.2+ is now also the only supported version of Ruby for Rails.
+
+## Please help us make Rails 5.0 solid!
+We rely on the feedback from everyone in the community to flush out bugs and upgrade issues ahead of a big release like this. So please give Rails 5.0 a try on your app, and if you're starting a new app today, you should probably use the beta1 for that, if you're just the least bit savvy with Rails.
+
+Issues can be recorded on the [Github issues tracker][issues].
+
+Already, [789 people have contributed][contributors] to this new release of Rails. Please do become one of them!
+
+Your dynamic release manager duo for Rails 5.0 is [Eileen M. Uchitelle][eileen] and [Sean Griffin][sean]. And the undisputed PR merge champ is [Rafael França][rafaelfranca]!
+
+[rails-1-0]: http://weblog.rubyonrails.org/2005/12/13/rails-1-0-party-like-its-one-oh-oh/
+[basecamp]: https://basecamp.com
+[jbuilder]: https://github.com/rails/jbuilder
+[ams]: https://github.com/rails-api/active_model_serializers
+[jsonapi]: https://github.com/cerebris/jsonapi-resources
+[turbolinks-v5]: https://github.com/basecamp/turbolinks/tree/v5
+[actioncable]: https://github.com/rails/rails/tree/master/actioncable
+[pratiknaik]: https://github.com/lifo
+[javanmakhmail]: https://github.com/javan
```

Comment:
You've got `mail`! But you probably want `mali` :grin:

Ditto in the Action Cable section.